freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/FreeIpaV1Endpoint.java

```diff
@@ -34,7 +34,8 @@
 import io.swagger.annotations.ApiOperation;
 
 @RetryAndMetrics
-@Path("/v1/freeipa")
+@Path("/" +
```

Comment:
I think this new line might not be intentional. could you revert this?

Reply:
Thanks a lot! Surprised this is not caught by style check. reverted it.